pootle/apps/pootle_statistics/models.py

```diff
@@ -563,8 +556,6 @@ def get_sugg_accepted():
             TranslationActionCodes.REVIEW_PENALTY: lambda: (-1) * reviewCost,
             TranslationActionCodes.SUGG_ACCEPTED: get_sugg_accepted,
             TranslationActionCodes.SUGG_REVIEWED_ACCEPTED: lambda: reviewCost,
-            TranslationActionCodes.SUGG_REJECTED: get_sugg_rejected,
-            TranslationActionCodes.SUGG_REVIEWED_REJECTED: lambda: analyzeCost,
```

Comment:
im not sure we are going to use in the future - but the scorelog entries that contained them would be auto removed